billboard.py

```diff
@@ -152,13 +185,16 @@ def __init__(self, name, date=None, fetch=True, max_retries=5, timeout=25):
             self.fetchEntries()
 
     def __repr__(self):
+        if self.year:
+            return "{}.{}({!r}, year={!r})".format(
+                self.__class__.__module__, self.__class__.__name__, self.name, self.year
+            )
         return "{}.{}({!r}, date={!r})".format(
             self.__class__.__module__, self.__class__.__name__, self.name, self.date
         )
 
     def __str__(self):
-        """Returns the chart as a human-readable string (typically multi-line).
-        """
+        """Returns the chart as a human-readable string (typically multi-line)."""
         if not self.date:
             s = "%s chart (current)" % self.name
```

Comment:
This needs to be updated. Maybe never show "current" for year-end charts?

```
>>> print(billboard.ChartData('country-catalog-albums-artists', year='2001'))
country-catalog-albums-artists chart (current)
----------------------------------------------
1. Taylor Swift
...
```

Reply:
Good catch, I'll make that change.